tools/mock_server/schemas/rum.py

```diff
@@ -27,12 +28,18 @@ class RUMSchema(Schema):
     event_jsons: [dict]
 
     def __init__(self, request: Request):
-        if request.headers.get('Content-Encoding', None) == 'deflate':
-            payload = zlib.decompress(request.get_data()).decode('utf-8')
+        self.headers = list(map(lambda h: f'{h[0]}: {h[1]}', request.headers))
+        self.data_as_text = request.get_data(as_text=True)
+
+        content_encoding = request.headers.get('Content-Encoding', None)
+        if content_encoding == 'deflate':
```

Comment:
I wonder where do we use `deflate` in Unity ? Currently we only use it for SR in the SDK and honestly it the way we compress it is a bit weird and it will not work to decompress each payload.